brtconfig.py

```diff
@@ -81,7 +81,7 @@ def get_docker_labels(self):
     From config file get the labels that should be added to the image
     """
 
-    config = module_framework.get_correct_config()
```

Comment:
If we're making this change, we should probably ensure that the tests are always guaranteed to use the newer version of MTF.

Either that, or do something like:
```
try:
    config = module_framework.get_config()
except AttributeError:
    # We're using an older version of MTF
    config = module_framework.get_correct_config()
```